llvm/lib/Target/RISCV/RISCVTargetTransformInfo.h

```diff
@@ -75,6 +75,22 @@ class RISCVTTIImpl : public BasicTTIImplBase<RISCVTTIImpl> {
                                       const APInt &Imm, Type *Ty,
                                       TTI::TargetCostKind CostKind);
 
+  /// \name Vector Predication Information
```

Comment:
There was a long comment re terminology here https://reviews.llvm.org/D99750#inline-1551705, but I cannot really tell how/if this still applies from the context.

Reply:
AS I said before, for 2 different objects we use different names.